components/policy/resources/templates/policy_definitions/README.md

```diff
@@ -0,0 +1,20 @@
+## Policy Settings in Brave
+
+These are Brave specific group policies.
+
+Our official documentation is [available here](https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy).
+
+For information about adding a new policy, please see [this page](https://source.chromium.org/chromium/chromium/src/+/main:docs/enterprise/add_new_policy.md).
+
+In a nutshell, the steps for adding a new policy in Brave look like this:
+
+1. Create a new .yaml file under `//brave/components/policy/resources/templates/policy_definitions/BraveSoftware/`. The name of the file itself will be the policy name. Chromium uses capital casing. For example, we have a policy for disabling Brave Rewards called `BraveRewardsDisable.yaml`. The name used for matching is `BraveRewardsDisable`.
+2. Update the properties in that file accordingly. Look at some of the existing ones as an example.
+3. Go into `//brave/browser/policy/brave_simple_policy_map.h` and add your entry here. It'll be auto-generated as `policy::key::k` and then the policy name. With the above example, that would be `policy::key::kBraveRewardsDisable`. You must map this to a profile preference (you must create a new one). That new preference is what you'll check in the code.
+4. Go into `//brave/chromium_src/components/policy/resources/policy_templates.py` and add your policy name to the end of the existing string array there.
+5. In the code where you want to check the profile preference, you can tell if it's set via policy by checking `prefs->IsManagedPreference(<profile_preference_key_here>)`. If this is set to true, you might want to have the UI display something like "This setting is managed by your organization" and have it be read-only.
+6. Build the project like regular (`npm run build`).
+7. You can build the policy templates using `npm run build -- --target=brave/components/policy:pack_policy_templates`.
+8. The group policy assets get output to `//out/<build_type_here>/brave_policy_templates.zip`. You can open this up and inspect the registry and adm/admx files.
```

Comment:
Is this still true? Or it is `policy_templates.zip`?

Reply:
Yup, this is true. Folks can use the `policy_templates.zip` - but those haven't had the script that renames Chrome => Brave run yet. The nice thing about `policy_templates.zip` is that it also has HTML documentation.

But yeah - we are currently sharing the `brave_policy_templates.zip` on the support page (just renamed as `policy_templates.zip`). Unless we make a change, that's the one we'll want to continue using

Reply:
Updated the README! 😄